### DISCUSSION TITLE
Firmware TASMOTA su esp8266 e12

### DISCUSSION THREAD

**Discussion by nervimarco** [indomus/forum#726] [Hardware & Firmware]
Salve, qualcuno è riuscito ad installare il firmware TASMOTA su un esp8266 e12?
Esiste una guida a riguardo?

Grazie mille

**Comment by indomus** [comment]
Dovrebbe bastarti la nostra [MASTERGUIDE](https://indomus.it/guide/riprogrammare-firmware-su-dispositivi-esp8266-sonoff-shelly-ecc-masterguide/).
